### PR TITLE
batch updated links in template files for easier use

### PR DESCRIPTION
**Pull request by MasonAdkins**
To make it easier moving forward I did a bunch of batch editing to get all of the template files usable within the project. This specifically address links to css, script, and image files. I checked through most of them but I don't want guarantee 100% accuracy of every link.

### CODE REVIEW

File: static/templates/browse-companies.html [hunk 1]
```diff
@@ -245,7 +245,7 @@ <h4>Messages</h4>
 										<!-- Notification -->
 										<li class="notifications-not-read">
 											<a href="dashboard-messages.html">
-												<span class="notification-avatar status-online"><img src="images/user-avatar-small-03.jpg" alt=""></span>
+												<span class="notification-avatar status-online"><img src="../../../../static/images/user-avatar-small-03.jpg" alt=""></span>
```
Comment (kfwojton) [inline static/templates/browse-companies.html]: this should likely just be ../../static not ../../../../static

Comment (MasonAdkins) [reply static/templates/browse-companies.html]: I used her original working index page as reference which had it that way. I just checked, seems to work with only '../../' so I adjusted all the img links to reflect that in another commit.

**Comment by MasonAdkins** [comment]
The final version of this PR leaves us with a entirely viewable version of the template as provided in the zip. My goal was to make development with this template easier having it all accessible for reference and further use.

**Comment by kfwojton** [comment]
Nice work! Thanks for replacing the static refs. As for the views/urls you made, you can simplify all of that in to one view and one url. You jsut have to do dynamic `template_name` references. 

url.py 

`  path('<template_name>/',  DynamicStaticPages.as_view()),` 

views.py 

```

class DynamicStaticPages(TemplateView):

    def get_context_data(self, **kwargs):

        data = super(DynamicStaticPages, self).get_context_data(**kwargs)
       #this is used to load only google analytics and hotjar on production server instead of testing/local or staging server. We don't want to get our activity to count in the analytics  
       data['server_name'] = os.getenv('SERVER_NAME')
 
        return data

    def get_template_names(self, **kwargs):

        if self.kwargs.get('template_name'):
            # favicons load first so with out this this dynamic view doesn't work. 
            if self.kwargs.get('template_name') == "favicon.ico":
                pass
            else:
                return self.kwargs.get('template_name') + ".html"
        #if there is no url default it index.html 

        return "index.html"
```

**Pull request merged**